tables/gurumuki.cti

```diff
@@ -20,191 +21,369 @@
 
 # This table is built and maintained under an activity of Braille Council of India
 # Contributors: Dipendra Manocha, Sreeja, Dinesh Kaushal, Mesar Hameed
-# Last updated on May 5, 2014
+# Updated on May 5, 2014
 # To report any bugs or any suggestion, please write to d@saksham.org and sreeja.param@gmail.com
 
+# Updated on May 12, 2022 by Jake Kyle, Compass Braille, jake@compassbraille.org
```

Comment:
This could be a bit confusing. I would add your name to the contributors, and there should be only one "updated" line I think. Also consider adding a computer-readable `#-updated: 2022-05-12` metadata field, and also `#author-name`, `#author-email`, `#-maintainer-name` and `#-maintainer-email` fields. If the original authors can indeed not be contacted anymore, I think you should become the maintainer. 

I know for sure Dipendra Manocha is still around and active though. Can you try to contact him one more time (with the email address that I will send you in a private message)?